awx/api/views/__init__.py

```diff
@@ -3858,8 +3858,7 @@ class JobJobEventsList(BaseJobEventsList):
     def get_queryset(self):
         job = self.get_parent_object()
         self.check_parent_access(job)
-        qs = job.job_events
-        qs = qs.select_related('host')
+        qs = job.job_events.select_related('host').order_by('start_line')
```

Comment:
![butwhy](https://user-images.githubusercontent.com/9753817/72750302-7d2f5100-3b8a-11ea-8ba8-37c2addfeef2.gif)

Reply:
https://github.com/ansible/awx/blob/03cfb7bf9ab29aff7c3262d0cd354d658324d530/awx/main/models/events.py#L429

?

Reply:
never mind, that explains nothing. Previously it was ordering by event, which is also indexed together with job.